### PR TITLE
Use discovery quick mode and multicall

### PR DESCRIPTION
**Pull request by michalsidzej**
Resolves L2B-2862

### CODE REVIEW

File: packages/backend/package.json
Comment (sdlyy) [inline packages/backend/package.json:1]: Double check packages, I think we have newer version of discovery release already, so let's keep package entries up to date

File: packages/backend/src/indexers/CurrentDiscoveryIndexer.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  readonly id = 'CurrentDiscoveryIndexer'
+
+  constructor(
+    private readonly blockNumberRepository: BlockNumberRepository,
+    private readonly discoveryRepository: DiscoveryRepository,
+    private readonly currentDiscoveryRepository: CurrentDiscoveryRepository,
+    private readonly indexerStateRepository: IndexerStateRepository,
+    private readonly chainId: ChainId,
+    logger: Logger,
+    discoverySnaphotIndexer: DiscoveryIndexer,
```
Comment (sdlyy) [inline packages/backend/src/indexers/CurrentDiscoveryIndexer.ts:22]: `snaphot` 🔥 😄 

Do we need to sync/differentiate old `DiscoveryIndexer` by renaming this one also to something like `DiscoverySnapshotIndxer`?

File: packages/backend/src/peripherals/database/migrations/008_current_discovery.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+If you find that something was incorrectly set up in the `up` function you
+should create a new migration file that fixes the issue.
+
+*/
+
+import { Knex } from 'knex'
+
+export async function up(knex: Knex): Promise<void> {
+  const discoveryRows = await knex('discovery').select('*')
+  console.log(discoveryRows)
```
Comment (sdlyy) [inline packages/backend/src/peripherals/database/migrations/008_current_discovery.ts:18]: clog

File: packages/backend/src/indexers/CurrentDiscoveryIndexer.ts (hunk 1 above)
Comment (michalsidzej) [reply packages/backend/src/indexers/CurrentDiscoveryIndexer.ts:22]: That was my idea, BUT the problem is that it has the `DiscoveryIndexer` ID in the DB. If we want to rename it, could be a separate PR

File: packages/backend/src/peripherals/database/migrations/008_current_discovery.ts (hunk 2 above)
Comment (michalsidzej) [reply packages/backend/src/peripherals/database/migrations/008_current_discovery.ts]: [58e4bbf](https://github.com/l2beat/lz-monitoring/pull/41/commits/58e4bbf5b1739e5942a997f4eb36ae2404779a21)

File: packages/backend/package.json
Comment (michalsidzej) [reply packages/backend/package.json:1]: [11788c7](https://github.com/l2beat/lz-monitoring/pull/41/commits/11788c707a8afd13159da9a0de2117d8e87ebf96)

**Pull request merged**